package.json

```diff
@@ -104,9 +104,6 @@
     "jest-cli": "^18.0.0",
     "node-fetch": "^1.6.3",
     "nyc": "^10.0.0",
-    "portfinder": "^1.0.10",
-    "react": "15.4.2",
-    "react-dom": "15.4.2",
```

Comment:
Please don't remove `react` and `react-dom`. Tests fail when you did clean install.